src/View/Helper/PaginatorHelper.php

```diff
@@ -14,6 +20,12 @@ class PaginatorHelper extends \Cake\View\Helper\PaginatorHelper
      */
     public function __construct(View $View, array $config = [])
     {
+        if (method_exists($View, 'getRequest')) {
```

Comment:
These are the downsides of forcing always latest minor via hard deprecations. :/